string      request_tag
bool        is_ball_detected
uint32      detected_x
uint32      detected_y
uint32      detected_r
